pkg/gotohelm/testdata/src/example/astrewrites/astrewrites.rewritten.go

```diff
@@ -0,0 +1,99 @@
+//go:build rewrites
+package astrewrites
```

Comment:
Where is the yaml representation of this file?

Reply:
This package is not currently being transpiled, it's just being used to showcase the AST rewriting. If you'd prefer, I could make all packages get transpiled and have their ASTs rewritten? It is a bit confusing to have everything live in the same place isn't it.

Reply:
Ok, now I understand.

Reply:
I went ahead and update this commit to perform rewrites on all the test packages and to transpile this one. Transpiling the rewrites package actually found a tiny bug that's fixed in the bootstrap commit, so it's disabled for this commit but will be enable in the bootstrap commit 👍